apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -69,37 +69,39 @@ const Ratings: React.FC<{ id: number; tutorName: string | null }> = ({
     );
 
   return (
-    <div className="flex flex-col my-8 mx-10 gap-10  justify-center">
-      {isEmpty(ratings) ? <NoTutorRatings tutorName={tutorName} /> : null}
+    <div className="flex flex-col my-8 mx-10 gap-8  justify-center">
+      {isEmpty(ratings) ? (
+        <NoTutorRatings tutorName={tutorName} />
```

Comment:
The spacing btween the image on the left and the image doesn't match the design. 

Try to zoom out the webpage and you will notice it doesn't match the design. 

The content should be centered with a fixed gap.